index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <link rel="stylesheet"  href="css/style.css">

```

Comment:
- [ ] Kindly make sure your CS file is correctly linked to the HTML file. as I can see they are on the root of the directory.
`Hint:` You may consider linking them with `<link="stylesheet" href="style.css"` and not the `css/style.css` you have in your HTML file.

![image](https://user-images.githubusercontent.com/97100091/205255067-e73084bb-8736-4f7a-83b6-a945be4c1929.png)